src/ColorShape.js

```diff
@@ -20,17 +23,18 @@ export default class ColorShape extends createjs.Shape {
 
     // Permanently change the color
     changeColor(color) {
-        this.drawSelf(color);
         this.color = color;
     }
 
-    resetColor() {
-        this.drawSelf(this.color);
+    // Permanently change the stroke thickness
+    changeStrokeThickness(thickness = DEFAULT_THICKNESS) {
+        this.thickness = thickness;
```

Comment:
prefer the convention set and get for field getters and setters i.e. setStrokeThickness()